docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -0,0 +1,363 @@
+# Summary Collection
+
+The core data structure that tracks summary attempts and corresponding results by monitoring the op log.
+
+## SummaryAckWithoutOp
+
+> Error
```

Comment:
It's not obvious that this means this event belong to `Error` category. Same goes with the other events. We can add the event types in the description of the doc and mention that each event belongs to one of these types.